### DISCUSSION TITLE
Why is PidTagLastVerbExecuted described here not same as MS-OXOMSG?

### DISCUSSION THREAD

**Discussion by dotfurther** [MicrosoftDocs/office-developer-client-docs#662] [Q&A]
Completely different PidTagLastVerbExecuted values from [MS-OXOMSG] spec:
https://docs.microsoft.com/en-us/openspecs/exchange_server_protocols/ms-oxomsg/87a8b6b8-59a4-4859-9dcd-8b0f36e3d729

Why is that?

**Comment by lindalu-MSFT** [comment]
@dotfurther Hi, I don't understand the question. Is there a bug in one of the topics in this docset? I'm not sure what you mean by "described here" because there is no link provided. Please provide a link to the topic and the steps to repro so I can troubleshoot. Thanks, lindalu

**Comment by dotfurther** [comment]
If I go to the MS link below, and press the "Edit" button, it takes me to this repository for the help.
If you compare the link below with the link above in my original post, the PidTagLastVerbExecuted is different than the [MS-OXOMSG] definition.

https://docs.microsoft.com/en-us/office/client-developer/outlook/mapi/pidtaglastverbexecuted-canonical-property

And my question is why the almost completely different PidTagLastVerbExecuted definitions between 2 Microsoft documents. One of them is wrong.

**Comment by dotfurther** [comment]
Pressing the "Edit" button at this MS link:
https://docs.microsoft.com/en-us/office/client-developer/outlook/mapi/pidtaglastverbexecuted-canonical-property
Takes me here, to this path, in this repository:
https://github.com/MicrosoftDocs/office-developer-client-docs/blob/master/docs/outlook/mapi/pidtaglastverbexecuted-canonical-property.md

And the definition HERE is not same as the [MS-OXOMSG] definition of PidTagLastVerbExecuted:
https://docs.microsoft.com/en-us/openspecs/exchange_server_protocols/ms-oxomsg/87a8b6b8-59a4-4859-9dcd-8b0f36e3d729

**Comment by lindalu-MSFT** [comment]
@dotfurther [This link](https://docs.microsoft.com/en-us/openspecs/exchange_server_protocols/ms-oxomsg/87a8b6b8-59a4-4859-9dcd-8b0f36e3d729) refers to exchange_server_protocols and [this link](https://github.com/MicrosoftDocs/office-developer-client-docs/blob/master/docs/outlook/mapi/pidtaglastverbexecuted-canonical-property.md) refers to Outlook 2013 client and Outlook 2016 client. What are you trying to do? Are you working in EWS or Outlook? What version? You could also ask your question to the developer community on Stack Overflow. Here is what I found when I searched for [PR_LAST_VERB_EXECUTED](https://stackoverflow.com/search?q=PR_LAST_VERB_EXECUTED).

**Comment by dotfurther** [comment]
I am working with Outlook client and PST saved.msg files. The PidTagLastVerbExecuted in Outlook client saved files follows the [MS-OXOMSG] definition.  PR_LAST_VERB_EXECUTED is just an alias for PidTagLastVerbExecuted per [MS-OXOPROPS]:
https://docs.microsoft.com/en-us/openspecs/exchange_server_protocols/ms-oxprops/4ec55eac-14b3-4dfa-adf3-340c0dcccd44

The Exchange master property list and the definition given here in this help repository are for the same property with ID  0x1081.  But their defined values do not match up.  It is the same MAPI property. It should have same definition for possible property values.

Why do you think the same MAPI property (0x1081) should have different property value definitions in different specifications?

**Comment by lindalu-MSFT** [comment]
I'm not sure why they're different, but there are many properties so maybe they BOTH are correct. The protocol documents have had many revisions over the years. I wouldn't know how to even begin to track down the answer, that's why I think StackOverflow developers might be able to help. Have you tried using a MAPI editor like MFCMapi or [OutlookSpy](http://www.dimastr.com/outspy/home.htm)? It will tell you if that property has been set or not (its an optional property set by the client). I also learned that PR_LAST_VERB_EXECUTED is only set on the messages in your mailbox. Are you using VBA? You could try a search for the DASL name [0x10810003](https://www.google.com/search?q=0x10810003&rlz=1C1GCEU_enUS821US821&oq=0x10810003&aqs=chrome..69i57.1053j0j7&sourceid=chrome&ie=UTF-8), you should be able to access it using [MailItem.PropertyAccessor.GetProperty](https://docs.microsoft.com/en-us/office/vba/api/outlook.propertyaccessor.getproperty). Also, [Referencing Properties by Namesake](https://docs.microsoft.com/en-us/office/vba/outlook/How-to/Navigation/referencing-properties-by-namespace) topic might be helpful.
